src/test/java/org/folio/rest/impl/PurchaseOrderLinesApiTest.java

```diff
@@ -203,9 +225,9 @@ public void testUpdateNonPackagePoLineWithoutTitle() throws MalformedURLExceptio
     assertThat(titleAfter.getProductIds(), hasSize(0));
     assertThat(titleAfter.getTitle(), is(newTitle));
 
-    deleteData(PURCHASE_ORDER.getEndpointWithId(), jsonOrder.getString("id"));
-    deleteData(PO_LINE.getEndpointWithId(), poLine.getId());
-    deleteData(TITLES.getEndpointWithId(), titleAfter.getId());
+    deleteData(PURCHASE_ORDER.getEndpointWithId(), jsonOrder.getString("id"), ISOLATED_TENANT_HEADER);
```

Comment:
There is no need to clean up data at the end of the test because it will be deleted by `IsolatedTenantExtension#afterTestExecution`.  This applies to all methods of this class.

Reply:
:unamused: